src/main/java/edu/csula/population/PopulationRecord.java

```diff
@@ -0,0 +1,63 @@
+package edu.csula.population;
+
+public class PopulationRecord {
+
+    private int total;
+
+    private int age;
+
+    private int females;
+
+    private int males;
+
+    private int year;
+
+    public PopulationRecord() {
+
+    }
+
+    public int getTotal() {
+        return total;
+    }
+
+    public void setTotal(int total) {
+        this.total = total;
+    }
+
+    public int getFemales() {
+        return females;
+    }
+
+    public void setFemales(int females) {
+        this.females = females;
+    }
+
+    public int getMales() {
+        return males;
+    }
+
+    public void setMales(int males) {
+        this.males = males;
+    }
+
+    public int getYear() {
+        return year;
+    }
+
+    public void setYear(int year) {
+        this.year = year;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+    @Override
+    public String toString() {
+        return String.format("Total: %d, Year: %d", total, year);
+    }
```

Comment:
As a thought, you guys can probably use Guava toString method https://github.com/google/guava/wiki/CommonObjectUtilitiesExplained#tostring to help debugging in future.